SU-SWS/decanter
# D8CORE-925: changing the icon to red from the grey

# READY FOR REVIEW
- Should be ok'ed by @kerri-augenstein as well. She ok'ed it in the screenshots from the ticket.

# Summary
- Updated the search icon color to red. 

# Needed By (Date)
- Feb 12

# Urgency
- Low. Accessibility issue.

# Steps to Test

1. Pull this branch.
2. Build.
3. View the top right search icon and verify it is red and not grey.
4. Make sure when using keyboard navigation that the focus is correct when on the search magnifying glass.

# Affected Projects or Products
- Our base theme will use it. 

# Associated Issues and/or People
- D8CORE-925
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- @kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
